README.md

```diff
@@ -136,9 +136,17 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Catima**](https://github.com/CatimaLoyalty/Android) <sup>**[[F-Droid](https://f-droid.org/app/me.hackerchick.catima)]**</sup>
 * [**QR & Barcode Scanner**](https://github.com/wewewe718/QrAndBarcodeScanner) <sup>**[[F-Droid](https://f-droid.org/app/com.example.barcodescanner)]**</sup>
 
+### • Battery
+
+* [**ACCA**](https://github.com/MatteCarra/AccA) <sup>**[F-Droid](https://f-droid.org/app/mattecarra.accapp)]**</sup>
+* [**Battery Tool**](https://github.com/Domi04151309/BatteryTool) <sup>**[[F-Droid](https://f-droid.org/app/io.github.domi04151309.batterytool)]**</sup>
+* [**BatteryBot Pro**](https://github.com/darshan-/Battery-Indicator-Pro) <sup>**[[F-Droid](https://f-droid.org/app/com.darshancomputing.BatteryIndicatorPro)]**</sup>
```

Comment:
Is this project still active?

Reply:
See my comment [here](https://github.com/MuntashirAkon/android-debloat-list/pull/29#discussion_r1303326164).